integration-tests/utils/integration-utils.js

```diff
@@ -86,11 +86,11 @@ const dismissNotifications = async page => {
 const signIntoTerra = async (page, token = bearerToken) => {
   await page.waitForXPath('//*[contains(normalize-space(.),"Loading Terra")]', { hidden: true })
   await waitForNoSpinners(page)
-  await page.evaluate(token => window.forceSignIn(token), token)
+  return page.evaluate(token => window.forceSignIn(token), token)
```

Comment:
TOL: Good catch. I think it worked before, but maybe it wasn't 100% "correct"? I'd like to learn more about this.

Reply:
My understanding it that it still worked but was actually taking a bit of unnecessary time. Awaiting the `page.evaluation` inside the function meant that the `signIntoTerra` function waited until that `page.evaluation` had a result and then since it's an async function returned the overall result wrapped in a promise so that in the integration tests we still had to `await signIntoTerra`, so essentially we end up double awaiting on the result of `page.evaluation`. Returning the `page.evaluation` instead directly passes that promise resolution back to the integration test which will await just once on the result there. (maybe @panentheos can weigh in if that is an accurate description of async/await though)

Reply:
Yes, that's right! In practice, any actual time overhead would be too tiny to matter, but it's still unnecessary so it seems like a good change. Note that this does change the signature of `signIntoTerra`, since its returned promise now resolves to something instead of resolving to `undefined`, but we weren't doing anything with the return value anyway so it should be ok.